src/main/java/com/liveramp/captain/daemon/CaptainJoblet.java

```diff
@@ -296,6 +305,10 @@ private void checkRequestComplete(CaptainRequestConfig config) {
         CaptainJoblet.of(newConfig, notifier, requestUpdater, manifestManager, supportsPending, rammingSpeed, failedRequestPolicy).run();
       }
 
+    } catch (CaptainTransientFailureException ce) {
+      String subject = String.format("%s: handle transient status retrieval for request %s", CaptainAlertHelpers.getHostName(), config.getId());
```

Comment:
Can we change the wording of all these exceptions to be something like: "%s: Transient failure while retrieving status for request %s. Doing nothing."